Tevling/Components/ChallengeCard.razor

```diff
@@ -23,18 +39,56 @@
         }
         else
         {
-            <div class="card-header @HeaderStyle d-flex justify-content-between">
-                <span>@Challenge.Measurement</span>
-                <span>@Challenge.Start.ToString("MMM")-@Challenge.End.ToString("MMM")</span>
-            </div>
-            <div class="card-body d-flex flex-column justify-content-between rg-2">
-                <div>
-                    <h5 class="card-title">@Challenge.Title</h5>
-                    <p class="card-text">@Challenge.Description</p>
+            @if (Challenge.End <= DateTimeOffset.Now)
```

Comment:
`DateTimeOffset.Now` will be server time and I guess client time is more accurate? Have a look at `BrowserTime` used elsewhere.